tests/scripts/scriptlib.fsx

```diff
@@ -77,7 +85,7 @@ module Scripting =
         if Directory.Exists output then 
             Directory.Delete(output, true) 
 
-    let log format = printfn format
+    let log format = fprintfn MessageSink.sinkWriter format
```

Comment:
logs diagnostic messages to the original non-redirected Console.Out. These will show up in xUnit's output.